src/app/error.tsx

```diff
@@ -1,18 +1,24 @@
+'use client';
```

Comment:
Not a server component?

Reply:
error page must be a client component https://nextjs.org/docs/app/api-reference/file-conventions/error